packages/cli-platform-apple/src/commands/buildCommand/createBuild.ts

```diff
@@ -12,7 +12,10 @@ import {supportedPlatforms} from '../../config/supportedPlatforms';
 const createBuild =
   ({platformName}: BuilderCommand) =>
   async (_: Array<string>, ctx: Config, args: BuildFlags) => {
-    const platformConfig = ctx.project[platformName] as IOSProjectConfig;
+    const platformConfig = ctx.project[platformName] as
+      | IOSProjectConfig
+      | undefined;
```

Comment:
I think we can also skip casting this type to `undefined`.

```suggestion
    const platformConfig = ctx.project[platformName] as IOSProjectConfig;
```